using-mobile-extensions/adobe-analytics-edge/README.md

```diff
@@ -0,0 +1,199 @@
+# Adobe Analytics Edge
+
+{% hint style="warning" %}
+
+The AEP Analytics Edge mobile extension is currently in BETA. Use of this extension is by invitation only. Please contact your Adobe Customer Success Manager to learn more.
+
+{% endhint %}
+
+## **Configure the Adobe Experience Platform Edge extension in** Experience Platform **Launch**
+
+1. In Experience Platform Launch, click the **Extensions** tab.
+2. On the **Catalog** tab, locate the **Adobe Experience Platform Edge** extension, and click **Install**.
+3. Within the Edge Extension configuration page add your Edge configuration and environment settings. For more information on creating an Edge configuration for use with your mobile app, see [Experience Platform Setup](../../beta/experience-platform-extension/experience-platform-setup). Additionally, Analytics Edge extension setup steps are available in a tutorial at [Enable Analytics in Edge configuration.](../../beta/experience-platform-extension/tutorials/tutorial-5-analytics-edge-extension#enable-analytics-in-edge-configuration)
+4. Click **Save**.
+5. Publish the latest changes to your **Adobe Experience Platform Launch** configuration and proceed to adding the Analytics Edge extension in your mobile app.
+
+## Add the Analytics Edge extension to your app
+
+{% tabs %}
+{% tab title="Android" %}
+**Java**
+
+1. Add the Mobile Core, Edge, and Analytics Edge extensions to your project using the app's Gradle file.
+
+   ```java
+    implementation 'com.adobe.marketing.mobile:sdk-core:1.+'
+    implementation 'com.adobe.marketing.mobile:edge:1.+'
+    implementation 'com.adobe.marketing.mobile:analyticsedge:1.+'
+   ```
+
+2. Import the AEP SDK extensions in your application's main activity.
+
+   ```java
+    import com.adobe.marketing.mobile.*;
+   ```
+{% endtab %}
+
+{% tab title="iOS" %}
+1. Add the Mobile Core, Edge, Identity, and Analytics Edge extensions to your project using Cocoapods.
+
+2. Add the following pods in your `Podfile`:
+
+   ```ruby
+    pod 'AEPCore'
+    pod 'AEPEdge'
+    pod 'AEPIdentity'
+    pod 'AEPAnalyticsEdge', :git => 'https://github.com/adobe/aepsdk-analyticsedge-ios', :branch => 'main' 
+   ```
+
+3. Import the Mobile Core, Edge, Identity, Analytics Edge, and AEP Services libraries:
+
+   **Objective-C**
+
+   ```text
+    @import AEPCore;
+    @import AEPEdge;
+    @import AEPIdentity;
+    @import AEPAnalyticsEdge;
+    @import AEPServices;
+   ```
+
+   **Swift**
+
+   ```swift
+    import AEPCore
+    import AEPEdge
+    import AEPIdentity
+    import AEPAnalyticsEdge
+    import AEPServices
+   ```
+   {% endtab %}
+
+   {% endtabs %}
+
+### Register Analytics Edge with Mobile Core
+
+{% tabs %}
+{% tab title="Android" %}
+#### Java
+
+To call the set up methods that call the [setApplication\(\)](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#setapplication) method in the `onCreate()` method:
```

Comment:
Let's rephrase this to something like:
In your Application class, register Analytics Edge with Mobile Core in the `onCreate()` method.